xml/System.Diagnostics/ActivitySpanId.xml

```diff
@@ -262,10 +262,12 @@ The characters in <paramref name="idData" /> are not all lower-case hexadecimal
         <Parameter Name="spandId2" Type="System.Diagnostics.ActivitySpanId" />
       </Parameters>
       <Docs>
-        <param name="spanId1">To be added.</param>
-        <param name="spandId2">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="spanId1">The first <see cref="T:System.Diagnostics.Tracing.ActivitySpanId" /> to compare.</param>
+        <param name="spandId2">The second <see cref="T:System.Diagnostics.Tracing.ActivitySpanId" /> to compare.</param>
+        <summary>Determines whether two specified ActivitySpanIds have the same value.</summary>
+        <returns>
+            <see langword="true" /> if the SpanId of <paramref name="spanId1" /> is the same as the SpanId of <paramref name="spandId2" />; otherwise, <see langword="false" />.
```

Comment:
nit: can you please put the whole returns in a single line? (GitHub doesn't allow multi-line suggestions, unfortunately):

```xml
<returns><see langword="true" /> if the SpanId of <paramref name="spanId1" /> is the same as the SpanId of <paramref name="spandId2" />; otherwise, <see langword="false" />.</returns>
```

Reply:
Same suggestion applies for the rest of the multi-line returns.